src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,169 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "abcdef123";
+    private static final String LENGTH_FOUR_INVALID_LOGIN = "1234";
+    private static final String LENGTH_EMPTY_INVALID_LOGIN = "";
+    private static final String LENGTH_SIX_VALID_LOGIN = "123456";
+    private static final String VALID_PASSWORD = "tazbog123";
+    private static final String LENGTH_FOUR_INVALID_PASSWORD = "abcd";
+    private static final String LENGTH_EMPTY_INVALID_PASSWORD = "";
+    private static final String LENGTH_SIX_VALID_PASSWORD = "abcdef";
+    private static final Integer VALID_AGE = 25;
+    private static final Integer YOUNG_INVALID_AGE = 17;
+    private static final Integer NEGATIVE_INVALID_AGE = -59;
+    private static final Integer MIN_VALID_AGE = 18;
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        user = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user.setLogin(VALID_LOGIN);
+        user.setPassword(VALID_PASSWORD);
+        user.setAge(VALID_AGE);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_validData_Ok() {
+        registrationService.register(user);
```

Comment:
It's better to create data in each test method. The main logic is: given (test data), when , then. 
User expected = new User - given
User actual = registrationService.register(expected); - when
assertEquals(expected, actual); - then 

Refactor you tests according to this logic